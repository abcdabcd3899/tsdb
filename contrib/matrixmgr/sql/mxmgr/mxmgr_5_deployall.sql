\c matrixmgr

SELECT proargnames, prosrc FROM pg_proc WHERE  proname = 'mxmgr_validate';

SELECT proargnames, prosrc FROM pg_proc WHERE  proname = 'mxmgr_deploy';

SELECT proargnames, prosrc FROM pg_proc WHERE  proname = 'mxmgr_remove_all';
