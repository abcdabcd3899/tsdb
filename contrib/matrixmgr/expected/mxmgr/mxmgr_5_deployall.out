\c matrixmgr
SELECT proargnames, prosrc FROM pg_proc WHERE  proname = 'mxmgr_validate';
  proargnames  |                                  prosrc                                  
---------------+--------------------------------------------------------------------------
 {schema_name} |                                                                         +
               | BEGIN                                                                   +
               |     PERFORM matrixmgr_internal.mxmgr_gate_deploy(schema_name, true);    +
               |     PERFORM matrixmgr_internal.mxmgr_telegraf_deploy(schema_name, true);+
               |     RETURN true;                                                        +
               | END;                                                                    +
               | 
(1 row)

SELECT proargnames, prosrc FROM pg_proc WHERE  proname = 'mxmgr_deploy';
  proargnames  |                                  prosrc                                   
---------------+---------------------------------------------------------------------------
 {schema_name} |                                                                          +
               | BEGIN                                                                    +
               |     PERFORM matrixmgr_internal.mxmgr_gate_deploy(schema_name, false);    +
               |     PERFORM matrixmgr_internal.mxmgr_telegraf_deploy(schema_name, false);+
               |     RETURN true;                                                         +
               | END;                                                                     +
               | 
(1 row)

SELECT proargnames, prosrc FROM pg_proc WHERE  proname = 'mxmgr_remove_all';
  proargnames  |                               prosrc                               
---------------+--------------------------------------------------------------------
 {schema_name} |                                                                   +
               | BEGIN                                                             +
               |     PERFORM matrixmgr_internal.mxmgr_remove_gate(schema_name);    +
               |     PERFORM matrixmgr_internal.mxmgr_remove_telegraf(schema_name);+
               |     RETURN true;                                                  +
               | END;                                                              +
               | 
(1 row)

